constitution.tex

```diff
@@ -477,17 +469,15 @@
 Directorship Name & Percentage of Dues \\
 \hline
 \hline
-Operational Communications & 20\% \\
-\hline
 Evaluations \& Selections & 5\% \\
 \hline
-House History & 10\% \\
+House History & 15\% \\
 \hline
-House Improvements & 15\% \\
+House Improvements & 20\% \\
 \hline
-Research and Development & 20\% \\
+Research and Development & 25\% \\
 \hline
-Social & 20\% \\
+Social & 25\% \\
```

Comment:
Can't conscionably advocate for increasing the social budget. As we all know, social bad.

Reply:
I think "Evil Board" (EBoard for short) would be more concise here

Reply:
@SpencerCreveling Before I bring this up at house meeting, I think you should add a line that defines EBoard as "Evil Board", preferably in as many commits as possible